packages/sdk/src/index.ts

```diff
@@ -75,9 +75,15 @@ export class MermaidChart {
   public async getAuthorizationData({
     state,
     scope = ['email'],
+    utm_source,
+    utm_medium,
+    utm_campaign
   }: {
     state?: string;
     scope?: string[];
+    utm_source?: string;
+    utm_medium?: string;
+    utm_campaign?: string;
```

Comment:
If you need all three present to send the tracking link, then it should be evident in the type. Otherwise the consumer of the SDK (some other plugin developer), might just send one, and be confused why it's not being sent, without looking inside the implementation. 

```suggestion
  public async getAuthorizationData({
    state,
    scope = ['email'],
    tracking
  }: {
    state?: string;
    scope?: string[];
    tracking?: {
      utm_source: string;
      utm_medium: string;
      utm_campaign: string;
    }
```